### PR TITLE
Added missing arg description in lraspp_mobilenet_v3_large

### PR DESCRIPTION
**Pull request by frgfm**
This PR adds a missing argument description for `lraspp_mobilenet_v3_large`.

Any feedback is welcome!

### CODE REVIEW

File: torchvision/models/segmentation/segmentation.py [hunk 1]
```diff
@@ -181,6 +181,7 @@ def lraspp_mobilenet_v3_large(pretrained=False, progress=True, num_classes=21, *
             contains the same classes as Pascal VOC
         progress (bool): If True, displays a progress bar of the download to stderr
         num_classes (int): number of output classes of the model (including the background)
+        aux_loss (bool): If True, it uses an auxiliary loss
```
Comment (NicolasHug) [inline torchvision/models/segmentation/segmentation.py:184]: thanks for the PR @frgfm 

If you look just below we have

```py
    if kwargs.pop("aux_loss", False):
        raise NotImplementedError('This model does not use auxiliary loss')
```

So documenting `aux_loss` is actually more confusing than anything. We should instead indicate how the `kwargs` argument is used

Comment (frgfm) [reply torchvision/models/segmentation/segmentation.py:184]: Oh my bad, I misread, I thought the arg was declared outside of kwargs!

Comment (NicolasHug) [reply torchvision/models/segmentation/segmentation.py:184]: We'll  be happy to consider a Pr for kwargs though, if you feel like it :) !

Comment (frgfm) [reply torchvision/models/segmentation/segmentation.py:184]: Mmmh, what do you have in mind?

Something like saying whose kwargs are those and mentioning the special case of aux_loss?

**Pull request closed without merging**